tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/NUnitEvpTests.cs

```diff
@@ -423,112 +610,109 @@ public async Task EarlyFlakeDetection(string packageVersion, string evpVersionTo
-                out var sessionCommand,
-                out var sessionWorkingDirectory,
-                out var gitRepositoryUrl,
-                out var gitBranch,
-                out var gitCommitSha);
-
-            try
-            {
-                using var agent = EnvironmentHelper.GetMockAgent();
-                agent.Configuration.Endpoints = agent.Configuration.Endpoints.Where(e => !e.Contains(evpVersionToRemove)).ToArray();
-
-                const string correlationId = "2e8a36bda770b683345957cc6c15baf9";
-                agent.EventPlatformProxyPayloadReceived += (sender, e) =>
-                {
-                    if (e.Value.PathAndQuery.EndsWith("api/v2/libraries/tests/services/setting"))
-                    {
-                        e.Value.Response = new MockTracerResponse(settingsJson, 200);
-                        return;
-                    }
-
-                    if (e.Value.PathAndQuery.EndsWith("api/v2/ci/libraries/tests"))
-                    {
-                        e.Value.Response = new MockTracerResponse(testsJson, 200);
-                        return;
-                    }
-
-                    if (e.Value.PathAndQuery.EndsWith("api/v2/ci/tests/skippable"))
-                    {
-                        e.Value.Response = new MockTracerResponse($"{{\"data\":[],\"meta\":{{\"correlation_id\":\"{correlationId}\"}}}}", 200);
-                        return;
-                    }
+            await ExecuteTestAsync(
+                    packageVersion,
+                    evpVersionToRemove,
+                    expectedGzip,
+                    new TestScenario(
+                        nameof(NUnitEvpTests),
+                        friendlyName,
+                        mockData,
+                        expectedExitCode,
+                        expectedSpans,
+                        true,
+                        (in ExecutionData data) =>
+                        {
+                            // Check the tests, suites and modules count
+                            Assert.Equal(ExpectedTestSuiteCount, data.TestSuites.Count);
+                            Assert.Single(data.TestModules);
+                        },
+                        useDotnetExec: false))
+               .ConfigureAwait(false);
+        }
 
-                    if (e.Value.PathAndQuery.EndsWith("api/v2/citestcycle"))
-                    {
-                        e.Value.Headers["Content-Encoding"].Should().Be(expectedGzip ? "gzip" : null);
+        [SkippableTheory]
+        [MemberData(nameof(GetDataForQuarantinedTests))]
+        [Trait("Category", "EndToEnd")]
+        [Trait("Category", "TestIntegrations")]
+        [Trait("Category", "QuarantinedTests")]
+        public async Task QuarantinedTests(string packageVersion, string evpVersionToRemove, bool expectedGzip, MockData mockData, int expectedExitCode, int expectedSpans, string friendlyName)
+        {
+            await ExecuteTestAsync(
```

Comment:
Previous tests here were marked skip alpine... should these ones be skipped too?